cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -30,10 +30,7 @@ hunter_test_string_not_empty("@HUNTER_PACKAGE_LICENSE_DIR@")
 hunter_test_string_not_empty("@HUNTER_GLOBAL_SCRIPT_DIR@")
 hunter_test_string_not_empty("@HUNTER_TLS_VERIFY@")
 
-if(APPLE)
-  set(configure_command "./Configure")
-  set(configure_opts "darwin64-x86_64-cc")
-elseif(ANDROID OR RASPBERRY_PI OR OPENWRT)
+if(ANDROID OR RASPBERRY_PI OR OPENWRT)
```

Comment:
Since build instructions changed please update `PACKAGE_INTERNAL_DEPS_ID` of OpenSSL package, details:
* https://docs.hunter.sh/en/latest/reference/user-modules/hunter_download.html